### PR TITLE
(docs) update README.md

### PR DESCRIPTION
**Pull request by vladdoster**
- correct spelling
- correct grammar
- reduce verbiage
- correct article agreement
- add punctuation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,19 +23,19 @@ The functionality in PowerShell Editor Services is already available in the foll
...
   - Real-time semantic analysis of scripts using PowerShell Script Analyzer
-- The Debugging Service simplifies interaction with the PowerShell debugger (breakpoints, variables, call stack, etc)
+- The Debugging Service simplifies interaction with the PowerShell debugger (breakpoints, variables, call stack, etc.)
 - The [$psEditor API](http://powershell.github.io/PowerShellEditorServices/guide/extensions.html) enables scripting of the host editor
 - A full, terminal-based Integrated Console experience for interactive development and debugging
 
 ## Usage
 
-Looking to integrate PowerShell Editor Services into your [Language Server Protocol](https://microsoft.github.io/language-server-protocol/) compliant editor or client? We support two ways of connecting.
+Are you looking to integrate PowerShell Editor Services into your [Language Server Protocol](https://microsoft.github.io/language-server-protocol/) compliant editor or client? We support two ways of connecting.
```
Comment (rjmholt) [inline README.md:32]:
```suggestion
If you're looking to integrate PowerShell Editor Services into your [Language Server Protocol](https://microsoft.github.io/language-server-protocol/) compliant editor or client, we support two ways of connecting.
```

**Review by rjmholt — APPROVED** [review]

**Comment by vladdoster** [comment]
@rjmholt Thank you for the approval. Is there something I need to do to get the CodeQL and CSharp CI jobs to get reported?

**Comment by andschwa** [comment]
> @rjmholt Thank you for the approval. Is there something I need to do to get the CodeQL and CSharp CI jobs to get reported?

Heh, no, we just needed to not require those as they don't run on doc updates.

**Comment by vladdoster** [comment]
@andschwa Is auto-merge stuck or am I naive about how it works? Maybe it is waiting for a release event and then it will be merged?

**Comment by andschwa** [comment]
Not naive, it was just stuck for some reason.

**Pull request merged**